libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -0,0 +1,318 @@
+
+    expect(fixture.nativeElement.querySelector('sky-icon')).toExist();
+    expect(fixture.nativeElement.querySelector('sky-icon-stack')).toBe(null);
+  });
+
+  it('should use sky-icon-stack in modern', async () => {
+    mockThemeSvc.settingsChange.next({
+      currentSettings: new SkyThemeSettings(
+        SkyTheme.presets.modern,
+        SkyThemeMode.presets.light,
+      ),
+      previousSettings: mockThemeSvc.settingsChange.value.currentSettings,
+    });
+    fixture.detectChanges();
+    await fixture.whenStable();
+
+    expect(fixture.nativeElement.querySelector('sky-icon')).toBeNull();
+    expect(fixture.nativeElement.querySelector('sky-icon-stack')).toExist();
+  });
+
+  it('should set aria label with labelText', async () => {
+    component.labelText = 'test component';
+
+    fixture.detectChanges();
+
+    await checkAriaPropertiesAndAccessibility(
+      'Show help content for test component',
+      null,
+      null,
+    );
+  });
+
+  it('should use aria label with labelText over deprecated ariaLabel input', async () => {
+    component.labelText = 'test component';
+    component.ariaLabel = 'deprecated';
+
+    fixture.detectChanges();
+
+    await checkAriaPropertiesAndAccessibility(
+      'Show help content for test component',
+      null,
+      null,
+    );
+  });
+
+  it('should set help popover when popoverContent input is set', async () => {
+    component.popoverContent = 'content';
+    fixture.detectChanges();
+
+    const { popoverHarness } = await getPopoverTestHarness();
+    await popoverHarness.clickPopoverButton();
+
+    await expect(
+      await (await popoverHarness.getPopoverContent()).getBodyText(),
+    ).toBe('content');
+  });
+
+  it('should render help popover title', async () => {
+    component.popoverTitle = 'title';
+    component.popoverContent = 'content';
+    fixture.detectChanges();
+
+    const { popoverHarness } = await getPopoverTestHarness();
+    await popoverHarness.clickPopoverButton();
+
+    await expect(
+      await (await popoverHarness.getPopoverContent()).getBodyText(),
+    ).toBe('content');
+    await expect(
+      await (await popoverHarness.getPopoverContent()).getTitleText(),
+    ).toBe('title');
+  });
+
+  it('should render help popover if helpContext is a templateRef', async () => {
+    component.popoverContent = component.popoverTemplate;
+    fixture.detectChanges();
+
+    const { popoverHarness } = await getPopoverTestHarness();
+    await popoverHarness.clickPopoverButton();
+
+    await expect(
+      await (await popoverHarness.getPopoverContent()).getBodyText(),
+    ).toBe('this is a template');
+  });
+
+  it('should set ariaControls to popover id if popover content is set', async () => {
+    component.popoverContent = 'content';
+    fixture.detectChanges();
+
+    const { popoverHarness } = await getPopoverTestHarness();
+    await popoverHarness.clickPopoverButton();
+
+    // without this call, the test fails, unsure why
```

Comment:
This is concerning, especially if it possibly affects our consumers' tests. If you remove the popover harness and just click on the help button directly, does the test work?